popup.js

```diff
@@ -0,0 +1,160 @@
+  },
+};
+
+const modelHeaders = document.createElement('div');
+const modeltTitiles = document.createElement('h2');
+const modeltTitiles2 = document.createElement('h2');
+const closeModels = document.createElement('button');
+
+modeltTitiles.nextSibling = closeModels;
+modeltTitiles.innerText = modelContanirArry.modelheader.modeltitile;
+modeltTitiles2.innerText = modelContanirArry.modelheader.modeltitileMob;
+closeModels.innerText = modelContanirArry.modelheader.closeModel;
+modelHeaders.appendChild(modeltTitiles);
+modelHeaders.appendChild(modeltTitiles2);
+modeltTitiles2.className = 'model-titile2';
+modelHeaders.appendChild(closeModels);
+modelHeaders.className = 'model-header';
+modeltTitiles.className = 'model-title';
+closeModels.className = 'close-model';
+// console.log(modelHeaders);
+
+// Model tag
+const modelTags = document.createElement('div');
+const tags1 = document.createElement('a');
+const tags2 = document.createElement('a');
+const tags3 = document.createElement('a');
+const tag1Image = document.createElement('img');
+const tag2Image = document.createElement('img');
+const tag3Image = document.createElement('img');
+
+// tags1.src=modelContanirArry.modeltag.tag1;
+tags1.appendChild(tag1Image);
+tag1Image.src = modelContanirArry.modeltag.tag1;
+tags2.appendChild(tag2Image);
+tag2Image.src = modelContanirArry.modeltag.tag2;
+tags3.appendChild(tag3Image);
+tag3Image.src = modelContanirArry.modeltag.tag3;
+tags1.className = 'model-tag-item';
+tags2.className = 'model-tag-item';
+tags3.className = 'model-tag-item';
+modelTags.appendChild(tags1);
+modelTags.appendChild(tags2);
+modelTags.appendChild(tags3);
+modelTags.className = 'model-tag';
+tag3Image.className = 'ruby-on-rail';
+
+const ht = document.querySelector('.model-container');
+const hb = document.querySelector('.bottom-container');
+
+// Image and Description part
+const modelBodys = document.createElement('div');
+const modelImages = document.createElement('img');
+const bodyTexts = document.createElement('p');
+const modelDescriptions = document.createElement('div');
+const seeLive = document.createElement('a');
+const seeSource = document.createElement('a');
+
+modelBodys.className = 'model-body';
+bodyTexts.className = 'model-text';
+modelDescriptions.className = 'model-description';
+bodyTexts.innerText = modelContanirArry.modelBody.modeltext;
+modelImages.style.height = '330px';
+modelImages.style.marginBottom = '35px';
+modelImages.className = 'model-image';
+modelImages.src = modelContanirArry.modelBody.bodyimage;
+seeLive.href = 'https://yohannesdagnachew.github.io/';
+seeLive.innerHTML = '<img src="image/Enable1.svg" />';
+seeSource.href = 'https://yohannesdagnachew.github.io/';
+seeSource.style.marginLeft = '10px';
```

Comment:
Please make sure that the second button text is `See source` with a GitHub icon. The link on the See Source button should take me to the Github repository of the website.
Currently, both button text is `See Live` and both take me to the website.

![image](https://user-images.githubusercontent.com/38088818/168419051-81e2b73c-d1f4-4d8d-bd3c-88c4f89a7d88.png)

Figma:
![image](https://user-images.githubusercontent.com/38088818/168419082-db6f6612-c8cb-4945-bfc0-94b239fb050c.png)

- Kindly work on that and you are good to go, you've had a long week. Otherwise, everything is fantastic! 👏🏾